pkgs/development/compilers/gcc/11/default.nix

```diff
@@ -27,6 +27,7 @@
 , cloog # unused; just for compat with gcc4, as we override the parameter on some places
 , buildPackages
 , libxcrypt
+, disableGdbPlugin ? enablePlugin
```

Comment:
Should be `disableGdbPlugin ? !enablePlugin` as well? That way we could get to 0 rebuilds I suspect.

Reply:
The comment suggests that this option should be named `enabledPlugins` instead.